console/console-init/ui/src/graphql-module/queries/iot_device.ts

```diff
@@ -7,31 +7,52 @@ import gql from "graphql-tag";
 import { IDeviceFilter, getInitialFilter } from "modules/iot-device";
 import { ISortBy } from "@patternfly/react-table";
 
-const RETURN_IOT_DEVICE_DETAIL = (iotproject: string, deviceId: string) => {
+const RETURN_IOT_DEVICE_DETAIL = (
+  iotproject: string,
+  deviceId: string,
+  resolver?: string
+) => {
+  const defaultResolver = `
```

Comment:
Rename reolver to queryResolver and defaultResolver to defaultQueryResolver for uniformity.